plugins/CoreHome/vue/src/Widget/Widget.vue

```diff
@@ -151,7 +151,11 @@ export default defineComponent({
           }
         }
 
-        return widget;
+        if (widget.uniqueId) {
+          result.parameters = { ...result.parameters, uniqueId: widget.uniqueId };
+        }
+
+        return result;
```

Comment:
Here previously we returned `widget` and now we are returning `result` variable. It seems as this was written incorrectly since the extra documentation from the report would otherwise never be used.